### PR TITLE
Add bounding box support

### PR DESCRIPTION
**Pull request by stephenkhess**
Includes support for post-validation transformations to get things into an ES representation

### CODE REVIEW

**Comment by stephenkhess** [comment]
Fixed #17

File: test/document/boundingbox.js [hunk 1]
```diff
@@ -0,0 +1,618 @@
...
+    doc.setBoundingBox(validBoundingBox);
+
+    t.deepEquals(doc.getBoundingBox(), validBoundingBox);
+    t.end();
+
+  });
+
+};
+
+module.exports.all = function (tape, common) {
```
Comment (orangejulius) [inline test/document/boundingbox.js]: these are some incredible tests :+1:

**Comment by orangejulius** [comment]
:+1: :+1:

**Comment by missinglink** [comment]
:+1:

**Pull request merged**